src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,693 @@
+"""The unpolarized time-like NLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from numpy import power as npp
```

Comment:
The comment to the other PR still holds: this is just the Python built-in `**` operator.

Built-ins are better because more elementary, and more familiar to the reader.
However, it is definitely not top-priority.

Reply:
I assume that Numba compiles `npp` and `**` to the same thing, including support for complex numbers. However, this can be verified with a very brief example :)

If I'm wrong, ignore the comment, of course ^^

Reply:
I just followed the usage as was done in space like case (for consistency) which also uses numpy.power.